Exiled.API/Features/Ragdoll.cs

```diff
@@ -152,12 +174,17 @@ public DateTime CreationTime
         }
 
         /// <summary>
-        /// Gets the <see cref="RoleTypeId"/> of the ragdoll.
+        /// Gets or sets the <see cref="RoleTypeId"/> of the ragdoll.
         /// </summary>
-        public RoleTypeId Role => NetworkInfo.RoleType;
+        public RoleTypeId Role
+        {
+            get => NetworkInfo.RoleType;
+            set => NetworkInfo = new(NetworkInfo.OwnerHub, NetworkInfo.Handler, value, NetworkInfo.StartPosition, NetworkInfo.StartRotation, NetworkInfo.Nickname, NetworkInfo.CreationTime);
+        }
 
         /// <summary>
-        /// Gets a value indicating whether or not the ragdoll has expired and SCP-049 is unable to revive it.
+        /// Gets a value indicating whether or not the ragdoll has expired and SCP-049 is unable to revive it if was not being targets.
```

Comment:
what

Reply:
it's because Scp049 is able to revive if it's was a previous target of Scp049 (with it's SenseAbility)